--- /home/antonius/mycms01.sql
+++ /home/antonius/mycms02.sql
 
 --
@@ -43,7 +43,7 @@
 INSERT INTO `CMSCONTENT` VALUES 
 (1,1050,1047),(5,1058,1056),(6,1059,1057),(7,1060,1057),(8,1061,1056),(9,1062,10
 57),(10,1063,1057),(11,1072,1070),(12,1073,1071),(13,1074,1066),(14,1075,1032),(
-15,1076,1032),(16,1077,1032),(17,1078,1032);
+15,1076,1032),(16,1077,1032),(17,1078,1032),(19,1080,1057);
 /*!40000 ALTER TABLE `CMSCONTENT` ENABLE KEYS */;
 UNLOCK TABLES;
 
 --
@@ -243,7 +243,8 @@
 22:46:24'),(109,1072,'d37b46dd-d237-4ca1-973d-7a0e31514b4c','2012-12-29 
 22:46:31'),(110,1050,'5a809261-ac3f-4fb3-882b-9eeb9ea805b5','2012-12-29 
 22:46:36'),(111,1050,'022474cd-ecae-43cb-a9f3-d511581a7651','2012-12-29 
-22:48:15');
+22:48:15'),(112,1080,'a0850da0-72ee-41b2-af0f-4070ba592880','2012-12-30 
+12:36:05');
 /*!40000 ALTER TABLE `CMSCONTENTVERSION` ENABLE KEYS */;
 UNLOCK TABLES;
 
@@ -698,8 +699,10 @@
 niseed Syrup',NULL,NULL,'2012-12-17 
 14:29:25',1055,NULL,'\0'),(1050,'\0',0,'9bc6ff74-4133-4ec7-918e-8f9782abecb8','C
 MS Store',NULL,NULL,'2012-12-17 14:29:25',1044,NULL,'\0'),

++
(1080,'\0',0,'a0850da0-72ee-41b2-af0f-4070ba592880','Test4',NULL,NULL,'2012-12-30 12:36:38',1055,NULL,''),
++

 (1062,'\0',0,'a1cf2791-6cd3-4eb5-8b61-e448c74a545c','Aniseed Syrup',NULL,NULL,'2012-12-17 23:32:26',1055,NULL,'\0'),
 (1060,'\0',0,'a2e15556-3083-42db-9bd8-03d443a63e51','Chang',NULL,NULL,'2012-12-14 
 14:05:17',1055,NULL,'\0'),(1059,'\0',0,'a3917a56-80f1-45be-857a-c78d69c345e0','C
 
 
@@ -2179,7 +2182,17 @@
 template=\"1069\" sortOrder=\"4\" createDate=\"2012-12-17T14:28:28\" 
 updateDate=\"2012-12-17T14:28:28\" nodeName=\"Register\" urlName=\"register\" 
 writerName=\"admin\" creatorName=\"admin\" path=\"-1,1050,1073\" isDoc=\"\" 
-/>');
+/>'),(1080,'a0850da0-72ee-41b2-af0f-4070ba592880','2012-12-30 
+12:36:38','<ProductPage id=\"1080\" parentID=\"1058\" level=\"3\" 
+writerID=\"0\" creatorID=\"0\" nodeType=\"1057\" template=\"1055\" 
+sortOrder=\"0\" createDate=\"2012-12-30T12:36:04\" 
+updateDate=\"2012-12-30T12:36:05\" nodeName=\"Test4\" urlName=\"test4\" 
+writerName=\"admin\" creatorName=\"admin\" path=\"-1,1050,1058,1080\" 
+isDoc=\"\"><productName>Test4 
+Product</productName><productPrice>32</productPrice><productDescription><![CDATA
+[<p>Test 4 
+Description</p>]]></productDescription><productImage>1078</productImage></Produc
+tPage>');
 /*!40000 ALTER TABLE `CMSPREVIEWXML` ENABLE KEYS */;
 UNLOCK TABLES;
 
 --
@@ -3435,7 +3448,13 @@
 L,NULL,'CMS 
 Store',NULL),(315,1050,'022474cd-ecae-43cb-a9f3-d511581a7651',30,NULL,NULL,NULL,
 '									Welcome 
 to the CMS Store!'),
++
(316,1080,'a0850da0-72ee-41b2-af0f-4070ba592880',34,NULL,NULL,'Test4 Product',NULL),
(317,1080,'a0850da0-72ee-41b2-af0f-4070ba592880',35,32,NULL,NULL, NULL),
(318,1080,'a0850da0-72ee-41b2-af0f-4070ba592880',36,NULL,NULL,NULL,'<p>Test 4 Description</p>'),
(319,1080,'a0850da0-72ee-41b2-af0f-4070ba592880',37,1078,NULL, NULL,NULL);
++
 /*!40000 ALTER TABLE `CMSPROPERTYDATA` ENABLE KEYS */;
 UNLOCK TABLES;
 
 --
@@ -9545,7 +9576,10 @@
 23:28:32'),(1077,'\0',-1,0,1,'-1,1077',51,'82601ced-17e2-4c69-91fb-d3139fe13807'
 ,'MacLeopard','b796f64c-1f99-4ffb-b886-4bf4bc011a9c','2012-12-17 
 23:30:05'),(1078,'\0',-1,0,1,'-1,1078',52,'359cd554-cfea-40eb-b79f-848c291cec12'
 ,'Commodore','b796f64c-1f99-4ffb-b886-4bf4bc011a9c','2012-12-17 23:31:20'),
++
(1080,'\0',1058,0,3,'-1,1050,1058,1080',3,'2c16c521-396e-49dc-9fa6-9760775eba0a','Test4','c66ba18e-eaf3-4cff-8a22-41b16d66a972','2012-12-30 12:36:04');
++
 /*!40000 ALTER TABLE `UMBRACONODE` ENABLE KEYS */;
 UNLOCK TABLES;
 
